kubernetes/pods/rabbitmq-helm/README.md

```diff
@@ -97,3 +119,521 @@ rabbitmqAmqpsSupport:
     ssl_options.verify                = verify_peer
     ssl_options.fail_if_no_peer_cert  = false
 ```
+
+# Updating from 1.6.1
```

Comment:
Can you put everything below this in another `.md` file named `updating_from_1.6.1.md` and either link to it or just leave it as another file, because this chart will be updated again and the question is, do you keep the 1.6.1 update here or do you delete it or what?  I think moving it out to another file now makes the main `README.md` clean and approachable.